### PR TITLE
Update: Improve table editing style

### PR DESCRIPTION
**Pull request by Xuefeng-Zhu**
image

* Make trigger clickable area bigger
* Add background color
* adjust position in navi-report

### CODE REVIEW

File: packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less [hunk 1]
```diff
@@ -4,6 +4,14 @@
  */
 
 .number-format-dropdown {
+  &__trigger {
+    padding: 0 5px;
+  }
+
+  .ember-basic-dropdown-trigger--below {
+    background-color: @navi-gray-300;
```
Comment (cythrawll) [inline packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less]: Can we do the bg color on hover?

File: packages/reports/app/styles/navi-reports/components/report-view.less [hunk 2]
```diff
@@ -105,3 +105,8 @@
         justify-content: flex-end;
     }
 }
+
+.number-format-dropdown__container {
```
Comment (balajimanoharan) [inline packages/reports/app/styles/navi-reports/components/report-view.less]: Why is this here? Shouldn't it be here: https://github.com/yahoo/navi/blob/a236f1798cc0c0f6750443995ad01bf901100b19/packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less

File: packages/visualizations/app/styles/navi-visualizations/visualizations/table.less [hunk 3]
```diff
@@ -89,7 +89,6 @@
 
     &__number-format-dropdown {
```
Comment (balajimanoharan) [inline packages/visualizations/app/styles/navi-visualizations/visualizations/table.less]: Why do we have three places where styles for `number-format-dropdown` can be defined?

File: packages/reports/app/styles/navi-reports/components/report-view.less (hunk 2 above)
Comment (Xuefeng-Zhu) [reply packages/reports/app/styles/navi-reports/components/report-view.less]: The dropdown is displaying correctly in visualization dummy app, but not in report dummy app, so I guess we should just apply this style to report

Comment (balajimanoharan) [reply packages/reports/app/styles/navi-reports/components/report-view.less]: This feels out of place in the `report-view`

Comment (Xuefeng-Zhu) [reply packages/reports/app/styles/navi-reports/components/report-view.less]: Yeah, It is kind of tricky since dropdown is rendered in a wormhole

File: packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less [hunk 4]
```diff
@@ -4,6 +4,20 @@
  */
 
 .number-format-dropdown {
+  align-self: center;
+
+  &__trigger {
+    padding: 0 5px;
```
Comment (jkusa) [inline packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less]: Can we make the padding `5px` so it aligns with the `input` ?

File: packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less [hunk 5]
```diff
@@ -4,6 +4,20 @@
...
+  }
+
+  &__trigger:hover {
+    background-color: @navi-gray-300;
```
Comment (jkusa) [inline packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less]: Can we use a lighter gray: `@navi-gray-200:`

File: packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less (hunk 4 above)
Comment (Xuefeng-Zhu) [reply packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less]: Do you mean `padding : 5px`?

File: packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less (hunk 5 above)
Comment (Xuefeng-Zhu) [reply packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less]: I tried it, but it looks worse than 300. And 300 is also used for export dropdown, so it is better to use 300 here

File: packages/reports/app/styles/navi-reports/components/report-view.less (hunk 2 above)
Comment (Xuefeng-Zhu) [reply packages/reports/app/styles/navi-reports/components/report-view.less]: work for me

**Comment by balajimanoharan** [comment]
Can we update the screenshot once the changes are made?

**Comment by Xuefeng-Zhu** [comment]
@balajimanoharan padding updated

File: packages/reports/app/styles/navi-reports/vendor/number-format-dropdown.less [hunk 6]
```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * These styles exist only to override the default styles that are included
+ * in the ember-tag-input addon
```
Comment (balajimanoharan) [inline packages/reports/app/styles/navi-reports/vendor/number-format-dropdown.less]: Can we update this comment?

Comment (Xuefeng-Zhu) [reply packages/reports/app/styles/navi-reports/vendor/number-format-dropdown.less]: updated
